backend/src/database/repositories/organizationRepository.ts

```diff
@@ -1579,13 +1581,13 @@ class OrganizationRepository {
         }
       }
 
-      segmentId = segment.id
+      segments = getSegmentSubprojectIds(segment)
```

Comment:
NIT: can we call this `segmentIds` instead? It's ambiguous if this has ids or the full object this way